generators/server/templates/src/main/docker/monitoring.yml.ejs

```diff
@@ -0,0 +1,40 @@
+<%#
+ Copyright 2013-2019 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+version: '3'
+services:
+    <%= baseName.toLowerCase() %>-prometheus:
+        image: <%= DOCKER_PROMETHEUS %>
+        volumes:
+            - ./prometheus/:/etc/prometheus/
+        command:
+            - '--config.file=/etc/prometheus/prometheus.yml'
+        ports:
+            - 9090:9090
+        network_mode: "host" # to test locally running service
+    <%= baseName.toLowerCase() %>-grafana:
+        image: <%= DOCKER_GRAFANA %>
+        volumes:
+            - ./grafana/provisioning/:/etc/grafana/provisioning/
+        environment:
+            - GF_SECURITY_ADMIN_PASSWORD=jhipster
+            - GF_USERS_ALLOW_SIGN_UP=false
+            - GF_INSTALL_PLUGINS=grafana-piechart-panel
+        ports:
+            - 3000:3000
+        network_mode: "host" # to test locally running service
```

Comment:
It does not work on Mac OS...

Reply:
I remember we had found a solution for mac os, maybe we can add it as a comment.

Reply:
I manage to make it work by removing the network_mode "host" and using host.docker.internal as the ip address

Reply:
yes that's the solution I think.